projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.ts

```diff
@@ -26,8 +26,8 @@ import { Observable, Subscription, combineLatest } from 'rxjs';
 import { tap, map } from 'rxjs/operators';
 
 import { SuggestedAddressDialogComponent } from '../../shipping-address/address-form/suggested-addresses-dialog/suggested-addresses-dialog.component'; // tslint:disable-line
-import { infoIconImgSrc } from '../../../../../ui/images/info-icon';
 import { Card } from '../../../../../ui/components/card/card.component'; // tslint:disable-line
+import { ICON_TYPES } from '../../../../../../cms-components';
```

Comment:
This causes an issue with the build you should import from `../../../../../../cms-components/misc/icon/index`